src/django/requirements.txt

```diff
@@ -1,3 +1,5 @@
+Django==1.10.3
```

Comment:
Hm. This feels super awkward since now the parent container isn't the source of the Django version, and any updates to that would be overridden by this. Not sure what a good solution would be though. A comment here reminding future us to also update the parent container if we update this _might_ help.